src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetectionPageFair.java

```diff
@@ -0,0 +1,39 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPageFairObject;
+import org.testng.annotations.Test;
+
+public class TestAdsDetectionPageFair extends TemplateNoFirstLoad {
+
+    @NetworkTrafficDump
+    @Test(
+        groups = "AdsDetectAdBlockPageFair",
+        dataProviderClass = AdsDataProvider.class,
+        dataProvider = "adsDetectionPageFair"
+    )
+    public void adsDetectAdBlock(Page page) {
+        String url = AdsPageFairObject.addPageFairDetectionParam(page);
```

Comment:
Instead of creating a static method I would pass a page as an argument to ads pagefair object constructor
(to follow convention with others ads objects)